docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md

```diff
@@ -50,21 +36,31 @@ To support nested IoT Edge scenarios, the container image for the connected regi
 ```azurecli
 az acr import \
   --name mycontainerregistry001 \
-  --source mcr.microsoft.com/acr/connected-registry:0.2.0
+  --source mcr.microsoft.com/acr/connected-registry:0.3.0
 ```
 
 To learn more about nested IoT Edge scenarios, please visit [Tutorial: Create a hierarchy of IoT Edge devices](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/iot-edge/tutorial-nested-iot-edge.md).
 
+## Import the IotEdge and API Proxy images into your registry
+
+Import the images azureiotedge-api-proxy, azureiotedge-agent, azureiotedge-hub to your private registry using the same command as above. 
```

Comment:
I see you use the scope-map is using the repository name like  "azureiotedge-api-proxy" "azureiotedge-agent" "azureiotedge-hub"

Can you update the import section to use import command?

az acr import -n mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-agent:20210609.5 -t azureiotedge-agent:20210609.5
az acr import -n mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-hub:20210609.5 -t azureiotedge-hub:20210609.5
az acr import -n mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev -t azureiotedge-api-proxy:9.9.9-dev

Reply:
Resolved